server.c

```diff
@@ -131,6 +132,16 @@ log_addr(const char* descr,
   #define TCP_FASTOPEN_SERVER_BIT_MASK 0x2
 #endif
 
+/* header state for the PROXYv2 header (for TCP) */
+enum pp2_header_state_type {
+	/* no header encounter yet */
+	pp2_header_none = 0,
+	/* read the static part of the header */
+	pp2_header_init,
+	/* read the full header */
+	pp2_header_done
```

Comment:
Because this is not a set of type identifiers, like the enum for like the RR type list is, I think the lowercase is fine. It is also in other places like that. So, I do not see why to change the code style. Thanks for the heads up, though.